src/main/java/com/amazon/opendistroforelasticsearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -353,6 +354,35 @@ public PrivilegesEvaluatorResponse evaluate(final User user, String action0, fin
         final Set<String> allIndexPermsRequired = evaluateAdditionalIndexPermissions(request, action0);
         final String[] allIndexPermsRequiredA = allIndexPermsRequired.toArray(new String[0]);
 
+        // handle resolve requests
+        if (action0.equals("indices:admin/resolve/index")) {
```

Comment:
Can we use ResolveIndexAction.NAME ?

Reply:
Why don't we use request type instead of request name.